docs/content/users/extend/custom-compose-files.md

```diff
@@ -76,3 +76,61 @@ When defining additional services for your project, we recommended following the
 [`ddev exec`](../usage/commands.md#exec), [`ddev ssh`](../usage/commands.md#ssh), and [`ddev logs`](../usage/commands.md#logs) interact with containers on an individual basis.
 
 By default, these commands interact with the `web` container for a project. All of these commands, however, provide a `--service` or `-s` flag allowing you to specify the service name of the container to interact with. For example, if you added a service to provide Apache Solr, and the service was named `solr`, you would be able to run `ddev logs --service solr` to retrieve the Solr container’s logs.
+
+## Third Party Services May Need To Trust `ddev-webserver`
+
+In some cases, a third-party service (`docker-compose.*.yaml`) may actually consume content from the `ddev-webserver` container. For example, a PDF generator like `gotenberg` might need to consume webserver content to create a PDF, or a testing service may need to do the same.
+
+In this case, the third-party service is by default not configured to trust the certificate authority used by DDEV (via `mkcert`) so you either have to use just HTTP between the two containers, or you have to make the third-party service trust the `mkcert` certificate authority.
+
+The simplest technique is just to use HTTP between the two containers. For example. the [`ddev-selenium-standalone-chrome`](https://github.com/ddev/ddev-selenium-standalone-chrome) service must consume content, and it just conducts interactions with the `ddev-webserver` by accessing `http://web`, see [setup](https://github.com/ddev/ddev-selenium-standalone-chrome/blob/main/config.selenium-standalone-chrome.yaml#L17). In this case, the `selenium-chrome` container just accesses the `web` container via HTTP.
+
+A second technique is to tell the third-party service to ignore HTTPS/TLS errors. For example, if the third-party service uses `curl`, it could use `curl --insecure https://web` or `curl --insecure https://<project>.ddev.site`.
```

Comment:
Use the proper noun of [cURL](https://en.wikipedia.org/wiki/CURL) before the command because words are fun!

```suggestion
A second technique is to tell the third-party service to ignore HTTPS/TLS errors. For example, if the third-party service uses cURL, it could use `curl --insecure https://web` or `curl --insecure https://<project>.ddev.site`.
```

This would be a nice place to demonstrate using cURL via CLI, PHP with `curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false)`, etc. with tabs for each major language DDEV supports.

Reply:
Agreed, but we're already SO FAR IN THE WEEDS :)